### PR TITLE
add inital website using material ui kit/template

### PR DESCRIPTION
**Pull request by kidadamu**
run yarn build then yarn start

### CODE REVIEW

File: capstone-app/package.json [hunk 1]
```diff
@@ -3,14 +3,43 @@
...
+    "@babel/preset-react": "7.10.1",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-plugin-prettier": "3.1.3",
+    "eslint-plugin-react": "7.20.0",
+    "gulp": "4.0.2",
+    "gulp-append-prepend": "1.0.8",
+    "prettier": "2.0.5"
+  },
+  "optionalDependencies": {
+    "typescript": "3.9.3"
```
Comment (peter-parque) [inline capstone-app/package.json:41]: nit - is there typescript in the project?

File: capstone-app/src/App.js [hunk 2]
```diff
@@ -1,26 +1,199 @@
...
+import React from "react";
+// nodejs library that concatenates classes
+import classNames from "classnames";
+// @material-ui/core components
+import { makeStyles } from "@material-ui/core/styles";
+
+// @material-ui/icons
+
+// core components
+import Header from "components/Header/Header.js";
```
Comment (peter-parque) [inline capstone-app/src/App.js:11]: nit - google style guides have imports in alphabetical order

File: capstone-app/src/App.js [hunk 3]
```diff
@@ -1,26 +1,199 @@
...
+          <WorkSection />
+        </div>
+      </div>
+      <Footer />
     </div>
   );
 }
 
-export default App;
+
```
Comment (peter-parque) [inline capstone-app/src/App.js:85]: we can reference old code by checking out older commits, so it doesn't need to stay here

File: capstone-app/src/assets/jss/material-kit-react/components/cardHeaderStyle.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import {
+  warningCardHeader,
+  successCardHeader,
+  dangerCardHeader,
+  infoCardHeader,
+  primaryCardHeader
```
Comment (dmbm0830) [inline capstone-app/src/assets/jss/material-kit-react/components/cardHeaderStyle.js:6]: nit- alphabetical order for imports

File: capstone-app/src/assets/jss/material-kit-react/components/customDropdownStyle.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+  infoColor,
+  infoBoxShadow,
+  successColor,
+  successBoxShadow,
+  warningColor,
+  warningBoxShadow,
+  dangerColor,
+  dangerBoxShadow,
+  roseColor,
+  roseBoxShadow
```
Comment (dmbm0830) [inline capstone-app/src/assets/jss/material-kit-react/components/customDropdownStyle.js:14]: nit- alphabetical order

File: capstone-app/package.json [hunk 6]
```diff
@@ -3,14 +3,43 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.10.0",
```
Comment (EmeraldBaldwin) [inline capstone-app/package.json:6]: Do you need all of these added dependencies?

File: capstone-app/src/assets/jss/material-kit-react/components/snackbarContentStyle.js [hunk 7]
```diff
@@ -0,0 +1,87 @@
+import {
```
Comment (EmeraldBaldwin) [inline capstone-app/src/assets/jss/material-kit-react/components/snackbarContentStyle.js:1]: Do you need all of these files? Do you have a snackbar in this app?

File: capstone-app/package.json (hunk 1 above)
Comment (kidadamu) [reply capstone-app/package.json:41]: It's an optional dependency for one of the other dependencies

**Pull request closed without merging**